tests/connection_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// connection_test.go provides some helpers for basic connectivity tests to Cloud SQL instances.
+package tests
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+)
+
+// proxyConnTest is a test helper to verify the proxy works with a basic connectivity test.
+func proxyConnTest(t *testing.T, connName, driver, dsn string, port int, dir string) {
+	ctx := context.Background()
+
+	var args []string
```

Comment:
Would we eventually need to add a `-credential_file` arg here to pass in service account credentials?

Reply:
At some point we probably want to add tests for different auth flow, but for now I'm just worried about maintaining parity with the existing tests.